Traceback (most recent call last):
  File "/home/pranavsatheesh/host_galaxies/py_files/merger_file_from_brahma_tree.py", line 343, in <module>
    WriteMergerFile(savePath,basePath, snapNum, minNdm=minNdm, minNgas=minNgas, 
  File "/home/pranavsatheesh/host_galaxies/py_files/merger_file_from_brahma_tree.py", line 177, in WriteMergerFile
    sub_lentype = il_brahma.groupcat.loadSubhalos_postprocessed(basePath,snapNum=snapNum,fields=['SubhaloLenType'])
  File "/home/pranavsatheesh/arepo_package/BRAHMA_python/groupcat.py", line 419, in loadSubhalos_postprocessed
    return loadObjects_postprocessed(basePath, snapNum, "Subhalo", "subgroups", fields)
  File "/home/pranavsatheesh/arepo_package/BRAHMA_python/groupcat.py", line 81, in loadObjects_postprocessed
    with h5py.File(gcPath_postprocessed(basePath, snapNum), 'r') as f:
  File "/blue/lblecha/pranavsatheesh/.conda/envs/hfrl/lib/python3.10/site-packages/h5py/_hl/files.py", line 561, in __init__
    fid = make_fid(name, mode, userblock_size, fapl, fcpl, swmr=swmr)
  File "/blue/lblecha/pranavsatheesh/.conda/envs/hfrl/lib/python3.10/site-packages/h5py/_hl/files.py", line 235, in make_fid
    fid = h5f.open(name, flags, fapl=fapl)
  File "h5py/_objects.pyx", line 54, in h5py._objects.with_phil.wrapper
  File "h5py/_objects.pyx", line 55, in h5py._objects.with_phil.wrapper
  File "h5py/h5f.pyx", line 102, in h5py.h5f.open
FileNotFoundError: [Errno 2] Unable to synchronously open file (unable to open file: name = '/orange/lblecha/aklantbhowmick/GAS_BASED_SEED_MODEL_UNIFORM_RUNS/L12p5n512/AREPO/SM5_DFD_3_TNG//groups_032_postprocessed/fof_subhalo_tab_032_postprocessed.0.hdf5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
Traceback (most recent call last):
  File "/home/pranavsatheesh/host_galaxies/py_files/merger_file_from_brahma_tree.py", line 343, in <module>
    WriteMergerFile(savePath,basePath, snapNum, minNdm=minNdm, minNgas=minNgas, 
  File "/home/pranavsatheesh/host_galaxies/py_files/merger_file_from_brahma_tree.py", line 177, in WriteMergerFile
    sub_lentype = il_brahma.groupcat.loadSubhalos_postprocessed(basePath,snapNum=snapNum,fields=['SubhaloLenType'])
  File "/home/pranavsatheesh/arepo_package/BRAHMA_python/groupcat.py", line 419, in loadSubhalos_postprocessed
    return loadObjects_postprocessed(basePath, snapNum, "Subhalo", "subgroups", fields)
  File "/home/pranavsatheesh/arepo_package/BRAHMA_python/groupcat.py", line 81, in loadObjects_postprocessed
    with h5py.File(gcPath_postprocessed(basePath, snapNum), 'r') as f:
  File "/blue/lblecha/pranavsatheesh/.conda/envs/hfrl/lib/python3.10/site-packages/h5py/_hl/files.py", line 561, in __init__
    fid = make_fid(name, mode, userblock_size, fapl, fcpl, swmr=swmr)
  File "/blue/lblecha/pranavsatheesh/.conda/envs/hfrl/lib/python3.10/site-packages/h5py/_hl/files.py", line 235, in make_fid
    fid = h5f.open(name, flags, fapl=fapl)
  File "h5py/_objects.pyx", line 54, in h5py._objects.with_phil.wrapper
  File "h5py/_objects.pyx", line 55, in h5py._objects.with_phil.wrapper
  File "h5py/h5f.pyx", line 102, in h5py.h5f.open
FileNotFoundError: [Errno 2] Unable to synchronously open file (unable to open file: name = '/orange/lblecha/aklantbhowmick/GAS_BASED_SEED_MODEL_UNIFORM_RUNS/L12p5n512/AREPO/SM5_LW10_LOWSPIN_RICH_DFD_3_TNG-SE_AGN0.01//groups_032_postprocessed/fof_subhalo_tab_032_postprocessed.0.hdf5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
Traceback (most recent call last):
  File "/home/pranavsatheesh/host_galaxies/py_files/merger_file_from_brahma_tree.py", line 343, in <module>
    WriteMergerFile(savePath,basePath, snapNum, minNdm=minNdm, minNgas=minNgas, 
  File "/home/pranavsatheesh/host_galaxies/py_files/merger_file_from_brahma_tree.py", line 177, in WriteMergerFile
    sub_lentype = il_brahma.groupcat.loadSubhalos_postprocessed(basePath,snapNum=snapNum,fields=['SubhaloLenType'])
  File "/home/pranavsatheesh/arepo_package/BRAHMA_python/groupcat.py", line 419, in loadSubhalos_postprocessed
    return loadObjects_postprocessed(basePath, snapNum, "Subhalo", "subgroups", fields)
  File "/home/pranavsatheesh/arepo_package/BRAHMA_python/groupcat.py", line 81, in loadObjects_postprocessed
    with h5py.File(gcPath_postprocessed(basePath, snapNum), 'r') as f:
  File "/blue/lblecha/pranavsatheesh/.conda/envs/hfrl/lib/python3.10/site-packages/h5py/_hl/files.py", line 561, in __init__
    fid = make_fid(name, mode, userblock_size, fapl, fcpl, swmr=swmr)
  File "/blue/lblecha/pranavsatheesh/.conda/envs/hfrl/lib/python3.10/site-packages/h5py/_hl/files.py", line 235, in make_fid
    fid = h5f.open(name, flags, fapl=fapl)
  File "h5py/_objects.pyx", line 54, in h5py._objects.with_phil.wrapper
  File "h5py/_objects.pyx", line 55, in h5py._objects.with_phil.wrapper
  File "h5py/h5f.pyx", line 102, in h5py.h5f.open
FileNotFoundError: [Errno 2] Unable to synchronously open file (unable to open file: name = '/orange/lblecha/aklantbhowmick/GAS_BASED_SEED_MODEL_UNIFORM_RUNS/L12p5n512/AREPO/SM5_LW10_LOWSPIN_RICH_DFD_3_TNG_AGN0.01//groups_032_postprocessed/fof_subhalo_tab_032_postprocessed.0.hdf5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
